src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -257,12 +259,23 @@ class TableView extends React.Component<TableViewProps> {
       return dataRow[column.key];
     }
 
-    const hasLinkField = eventView.hasAutolinkField();
-    const forceLink =
-      !hasLinkField && eventView.getFields().indexOf(String(column.field)) === 0;
+    // TODO: remove this
+    // const hasLinkField = eventView.hasAutolinkField();
+    // const forceLink =
+    //   !hasLinkField && eventView.getFields().indexOf(String(column.field)) === 0;
+    const forceLink = false;
 
     const fieldRenderer = getFieldRenderer(String(column.key), tableData.meta, forceLink);
-    return fieldRenderer(dataRow, {organization, location});
+    return (
+      <ExpandAggregateRow
```

Comment:
This seems like something that could fit within the field context menus that we've been planning to do for while.

Reply:
@markstory I touched based with @doralchan on the field context menus; it sounds like it may have been deferred in favour of field links. 

It might be better for the work for the field context menus be in a separate PR.